FinSearchUnified/Tests/Bundles/SearchBundleDBAL/QueryBuilderFactoryTest.php

```diff
@@ -0,0 +1,272 @@
+<?php
+
+namespace FinSearchUnified\Tests\Bundles\SearchBundleDBAL;
+
+use FinSearchUnified\Bundles\SearchBundleDBAL\QueryBuilderFactory;
+use Shopware\Bundle\SearchBundle\Condition\CloseoutCondition;
+use Shopware\Bundle\SearchBundle\Criteria;
+use Shopware\Bundle\StoreFrontBundle\Service\ContextServiceInterface;
+use Shopware\Components\Test\Plugin\TestCase;
+
+class QueryBuilderFactoryTest extends TestCase
+{
+    /** @var QueryBuilderFactory */
+    private $criteriaFactory;
+
+    /** @var ContextServiceInterface $contextService */
```

Comment:
- Split this into multiple lines.
- Remove the variable name.

Reply:
@ihanli what do you mean split this into multiple lines? the declaration is already in the next line after the variable name comment.